backend-api/src/functions/asyncIssueBiometricCredential/asyncIssueBiometricCredentialHandler.test.ts

```diff
@@ -1049,94 +1052,348 @@ describe("Async Issue Biometric Credential", () => {
         });
       });
     });
-
-    describe("Happy path", () => {
+    describe("Given sending DCMAW_ASYNC_CRI_END event fails", () => {
       beforeEach(async () => {
+        dependencies = {
+          ...dependencies,
+          getEventService: () => mockFailingCriEventService,
+        };
+
         await lambdaHandlerConstructor(dependencies, validSqsEvent, context);
       });
 
-      it("Passes correct arguments to get secrets", () => {
-        expect(mockGetSecretsSuccess).toHaveBeenCalledWith({
-          secretNames: ["mockBiometricViewerAccessKey"],
-          cacheDurationInSeconds: 900,
+      it("Logs the DCMAW_ASYNC_CRI_END event failure", () => {
+        expect(consoleErrorSpy).toHaveBeenCalledWithLogFields({
+          messageCode: "MOBILE_ASYNC_ERROR_WRITING_AUDIT_EVENT",
+          data: {
+            auditEventName: "DCMAW_ASYNC_CRI_END",
+          },
         });
       });
 
-      it("Passes correct arguments to get biometric session", () => {
-        expect(mockGetBiometricSessionSuccess).toHaveBeenCalledWith(
-          "mockReadIdBaseUrl",
-          mockBiometricSessionId,
-          "mockViewerKey",
-        );
+      it("Does not log COMPLETED", () => {
+        expect(consoleInfoSpy).not.toHaveBeenCalledWithLogFields({
+          messageCode: "MOBILE_ASYNC_ISSUE_BIOMETRIC_CREDENTIAL_COMPLETED",
+        });
       });
+    });
 
-      it("Passes correct arguments to getCredentialFromBiometricSession", () => {
-        expect(
-          mockSuccessfulGetCredentialFromBiometricSession,
-        ).toHaveBeenCalledWith(
-          { finish: "DONE" },
-          {
-            userSessionCreatedAt: 1704106860000,
-            opaqueId: "mockOpaqueId",
-          },
-          {
-            enableUtopiaTestDocument: true,
-            enableDrivingLicence: true,
-            enableNfcPassport: true,
-            enableBiometricResidencePermit: true,
-            enableBiometricResidenceCard: true,
-          },
-        );
-      });
+    describe("Happy path", () => {
+      describe("Without flags or contraIndicators", () => {
+        const mockAuditData = {
+          contraIndicatorReasons: [],
+          txmaContraIndicators: undefined,
+          flags: null,
+          flaggedRecord: null,
```

Comment:
```suggestion
          contraIndicatorReasons: [],
          txmaContraIndicators: [],
```

^ this is what the data would actually look like